bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/ProgramId.scala

```diff
@@ -21,10 +21,10 @@ sealed trait ProgramId {
   def shortName: String =
     this match {
       case ProgramId.Science(site, semester, ptype, index) =>
-        s"${site.abbreviation.drop(1)}-${semester.toShortString}-${ptype.abbreviation}-$index"
+        s"${site.abbreviation.drop(1)}${semester.toShortString}-${ptype.abbreviation}-$index"
 
       case ProgramId.Daily(site, ptype, year, month, day) =>
-        f"${site.abbreviation.drop(1)}-${ptype.abbreviation}${year.toString.takeRight(2)}$month%02d$day%02d"
+        f"${site.abbreviation.drop(1)}${year.toString.takeRight(2)}$month%02d$day%02d-${ptype.abbreviation}"
```

Comment:
It wasn't clear what to do about the short name for `Daily` program ids.  Maybe it isn't an issue. We could just shorten say `GS-ENG20180320` to `SENG180320`, but I went with rearranging it a bit to correspond to the science ids: `S180320-ENG`.  Maybe this never comes up in the QPT.